### ISSUE TITLE
Loss function for Pre-training

### ISSUE THREAD

**Issue by blengerich** [cnellington/Contextualized#7]
It would be nice to have a loss function for pretraining which takes in prior knowledge of correlated/uncorrelated/anti-correlated pairs.
